test/tst_fixedrangescaler.jl.orig

```diff
@@ -0,0 +1,302 @@
+X = collect(Float64, reshape(1:40, 10, 4))
```

Comment:
This file seems to be an artefact from the rebase and should be deleted before merging